### ISSUE TITLE
No mention of ERC ?

### ISSUE THREAD

**Issue by f6fvy** [MalphasWats/hawk#1]
Nice job, but you don't mention (and don't use) the ERC (Electrical Rules Checking) in schematics. Which it pretty useful to detect obvious mistakes.

BTW, it would probably fail because you don't use any power flag.

**Comment by MalphasWats** [comment]
You're right actually - I do use it usually, I'm not sure why I didn't cover it. I will try and add a bit about it at some point.

The DRC check does pass for this board, and there are power flags on there.
